code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -590,9 +619,42 @@
 	required_reagents = list(/datum/reagent/silver = 20, /datum/reagent/carbon = 10)
 	required_temp = 630
 	mob_react = FALSE
+	reactionFlags = REACTION_INSTANT 
 
 /datum/chemical_reaction/silver_solidification/on_reaction(datum/reagents/holder, created_volume)
 	var/location = get_turf(holder.my_atom)
 	for(var/i in 1 to created_volume)
 		new /obj/item/stack/sheet/mineral/silver(location)
 
+/datum/chemical_reaction/basic_buffer
+	results = list(/datum/reagent/basic_buffer = 5)
+	required_reagents = list(/datum/reagent/lye = 1, /datum/reagent/consumable/ethanol = 2, /datum/reagent/water = 2)
+	//FermiChem vars:
+	required_temp 	= 250
+	optimal_temp 	= 500
+	overheat_temp  	= 9999 
+	optimal_pH_min 	= 0
+	optimal_pH_max 	= 14
+	determin_pH_range 		= 0
+	temp_exponent_factor 	= 4
+	pH_exponent_factor 	= 0
+	thermic_constant= 0
+	H_ion_release 	= 0.01
+	rate_up_lim 		= 15
+	purity_min		= 0
```

Comment:
This is a request not only here. Could you not add all these formatting spaces between the variable and the value?
They don't necessarily align properly depending on where they're read, and they make searches more difficult.

Reply:
I've done this for all instances, I'll check over the diff again to be sure I got them all.